tests/timelimit3-wf.cwl

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env cwl-runner
 class: Workflow
-cwlVersion: v1.1.0-dev1
+cwlVersion: v1.1
 
 requirements:
   ToolTimeLimit:
     timelimit: 0
   WorkReuse:
     enableReuse: false
+  InlineJavascriptRequirement: {}
```

Comment:
Good catch! The tags in `conformance_tests.yaml` will need to be updated as well